File: data/src/main/java/com/taxapprf/data/remote/cbrapi/CbrAPI.java
package com.taxapprf.data.remote.cbrapi;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface CbrAPI {

    @GET("XML_daily.asp")
    Call<Currencies> loadCurrencies(@Query("date_req") String date);
}

File: data/src/main/java/com/taxapprf/data/remote/cbrapi/CBRAPI.kt
package com.taxapprf.data.remote.cbrapi

import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface CBRAPI {
    @GET("XML_daily.asp")
    fun getCurrency(
        @Query("date_req") date: String
    ): Call<Currency>
}